gatorgrade/main.py

```diff
@@ -0,0 +1,32 @@
+"""Use Typer to run gatorgrade to run the checks and generate the yml file."""
+import typer
+from pathlib import Path
+
+app = typer.Typer(add_completion=False)
+FILE_DIR = "config"
```

Comment:
In my understanding, the `gatorgrade.yml` file should be at the root level of the assignment, not inside a `config` directory -- @mariakimheinert, as a potential future user of this project, is this true?

Reply:
Yes, this is true--`gatorgrade.yml` should be at the root level of the assignment.

Reply:
Ok! I changed this so that the .yml file will be generated at the root level.